src/lib.rs

```diff
@@ -499,29 +546,50 @@ impl GameWrapper {
 
 #[wasm_bindgen]
 pub fn make_game() -> Result<GameWrapper, JsValue> {
+    let CanvasInfo {
+        context,
+        width,
+        height,
+    } = get_canvas_info()
+        .map_err(|e| JsValue::from(format!("Error initializing canvas: {}", e)))?;
+    let ws = WebSocket::new("ws://localhost:3012")?;
+
     let mut players = vec![
         make_player(Color::Red),
         make_player(Color::Pink),
         make_player(Color::Blue),
         make_player(Color::Orange),
         make_player(Color::White),
         make_player(Color::Black),
+        make_player(Color::Green),
     ];
     let num_players = players.len() as f64;
+    // Pick one at random to be the impostor.
+    players[(js_sys::Math::random() * num_players).floor() as usize].impostor = true;
+
     for (i, player) in players.iter_mut().enumerate() {
         // Place the players equidistant around the meeting table.
         player.position = Position {
             x: 275.0 + (100.0 * ((i as f64) / num_players * 2.0 * f64::consts::PI).sin()),
             y: 275.0 + (100.0 * ((i as f64) / num_players * 2.0 * f64::consts::PI).cos()),
         };
+        // Give everyone 6 random tasks.
+        for _ in 0..6 {
+            player.tasks.push(Task {
+                position: Position {
+                    x: (js_sys::Math::random() * width)
```

Comment:
Is there no standard Rust idiom for this?

Reply:
For which? Generating a random number between two others? For that https://docs.rs/rand/0.7.3/rand/trait.Rng.html#method.gen_range but we'd need to use that crate and I thought that it failed to compile targeting wasm but now that I'm looking at the docs it sounds like it should actually work...